sources/packages/backend/apps/api/src/auth/auth.module.ts

```diff
@@ -49,6 +50,7 @@ const jwtModule = JwtModule.register({
     SINValidationService,
     RestrictionService,
     DesignationAgreementLocationService,
+    StudentRestrictionsService,
```

Comment:
This can be removed. Please ensure that the providers list contains only the below items.
```ts
UserService,
InstitutionLocationService,
InstitutionUserAuthService,
TokensService,
KeycloakService,
JwtStrategy,
StudentService,
SFASIndividualService,
SINValidationService,
DesignationAgreementLocationService,
```